masschallenge/semantic-ui-theme
# AC-6147: Update SUI site.variables file with MC Styles

**Changes introduced in AC-6147**:
- add SUI site variables

**How to test**:
- Edit the file src/themes/masschallenge/elements/button.variables and add `@verticalMargin: @primaryColor;`
- run `gulp build`
- check `dist/semantic.css` and search for `#0eacbd` (the value of the @primaryColor variable)
- finding this colour in a misplaced style will prove that were making use of our new SUI variables
